guest/go.mod

```diff
@@ -0,0 +1,13 @@
+module sigs.k8s.io/kube-scheduler-wasm-extension/guest
+
+go 1.19
```

Comment:
Can we use 1.20 like other modules?

Reply:
I think we should switch this when tinygo 0.28 is out cc @deadprogram https://github.com/tinygo-org/tinygo/milestone/14

Reply:
Ah, limitation via tinygo side. OK, that makes sense.